### PR TITLE
Create organizations.md

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: community/organizations.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+* [JuliaSparse](https://github.com/JuliaSparse) – Sparse matrix solvers
+* [SciML (formerly JuliaDiffEq)](https://github.com/SciML) – [Open source software for scientific machine learning](https://sciml.ai) ([Gitter](https://gitter.im/JuliaDiffEq/Lobby))
+@@
+
+### Scientific Domains
+
+@@tight-list
+* [BioJulia](https://github.com/BioJulia) – Biology, bioinformatics, computational biology ([website](https://biojulia.net), [Gitter](https://gitter.im/BioJulia/home))
+* [EcoJulia](https://github.com/EcoJulia) – Ecology
+* [JuliaAstro](https://github.com/JuliaAstro) – [Astronomy](https://juliaastro.github.io/) ([Gitter](https://gitter.im/JuliaAstro/home))
```
Comment (giordano) [inline community/organizations.md:49]: We don't really use gitter for JuliaAstro anymore

**Comment by logankilpatrick** [comment]
@DilumAluthge

File: community/organizations.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+* [JuliaRegistries](https://github.com/JuliaRegistries)
+* [JuliaCon](https://github.com/JuliaCon) - The Julia Conference
+* [JuliaAcademy](https://github.com/JuliaAcademy) - The definitive source for learning all things Julia.
+* [JuliaCommunity](https://github.com/JuliaCommunity) - Organizing Julia Community Initiatives
+@@
+
+### General
+
+@@tight-list
+* [JuliaDocs](https://github.com/juliadocs) – Documentation-related packages for Julia ([Gitter](https://gitter.im/juliadocs/users))
```
Comment (fredrikekre) [inline community/organizations.md:18]: Are you still monitoring Gitter @mortenpi?

Comment (mortenpi) [reply community/organizations.md:18]: I think I still get email notifications, but I wouldn't say that it's the preferred way of getting support on Documenter or JuliaDocs. #documentation in Slack or Zulip, or Discourse would be better.

**Pull request merged**